Makefile

```
@@ -1,5 +1,5 @@
 HOME=$(shell pwd)
-MAINVERSION=2.4
+MAINVERSION?=2.4
 NO_SUDO?=0
 USE_PODMAN?=0
 LUA_VERSION=5.4.3
```

```suggestion
LUA_VERSION?=5.4.3
```

Also allow for `LUA_VERSION` overrides?